apps/rush-lib/src/logic/pnpm/PnpmLinkManager.ts

```diff
@@ -290,6 +305,116 @@ export class PnpmLinkManager extends BaseLinkManager {
     });
   }
 
+  /**
+   * This is called once for each local project from Rush.json.
+   * @param project             The local project that we will create symlinks for
+   * @param rushLinkJson        The common/temp/rush-link.json output file
+   */
+  private async _linkWorkspaceProject(
```

Comment:
This is just generating the link status file, right? It isn't actually performing the linking?

Can you document that?

Reply:
I've already talked to Pete on this one, I think in a future PR we will refactor this into the install process itself and no-op/warn/error on `rush link` so that this isn't deceptive. I'll add a TODO comment